app/assets/javascripts/editor.js

```diff
@@ -127,6 +127,37 @@ function create_editor() {
   }
 }
 
+function set_fullscreen(){
```

Comment:
@alongoldboim IIRC this was needed before we started to use the Ace editor, it doesn't look like it's used anywhere now